manifests/service.pp

```diff
@@ -34,4 +34,12 @@
     status     => 'service pulp_resource_manager status | grep "node resource_manager"',
   }
 
+  service { 'pulp_streamer':
+    ensure     => running,
+    require    => [Service[mongodb]],
```

Comment:
Looks like we can add this to (https://github.com/Katello/puppet-pulp/blob/master/manifests/database.pp#L14-L17) and remove these requires. I agree lets do a follow up with that change.